app/src/main/java/com/github/onlynotesswent/ui/screen/EditNote.kt

```diff
@@ -1,23 +1,89 @@
 package com.github.onlynotesswent.ui.screen
 
+import android.graphics.Bitmap
+import androidx.compose.foundation.layout.Arrangement
 import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.padding
 import androidx.compose.material3.Button
+import androidx.compose.material3.OutlinedTextField
 import androidx.compose.material3.Text
 import androidx.compose.runtime.Composable
+import androidx.compose.runtime.collectAsState
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
 import androidx.compose.ui.platform.testTag
+import androidx.compose.ui.unit.dp
+import com.github.onlynotesswent.model.note.Note
 import com.github.onlynotesswent.model.note.NoteViewModel
+import com.github.onlynotesswent.model.note.Type
 import com.github.onlynotesswent.ui.navigation.NavigationActions
 import com.github.onlynotesswent.ui.navigation.Screen
+import com.google.firebase.Timestamp
 
+/**
+ * Displays the edit note screen, where users can update the title and content of an existing note.
+ * The screen includes two text fields for editing the note's title and content, and a button to
+ * save the changes. The save button updates the note in the ViewModel and navigates back to the
+ * overview screen.
+ *
+ * @param navigationActions The navigation view model used to transition between different screens.
+ * @param noteViewModel The ViewModel that provides the current note to be edited and handles note
+ *   updates.
+ */
 @Composable
 fun EditNoteScreen(navigationActions: NavigationActions, noteViewModel: NoteViewModel) {
```

Comment:
I would personally also add a go back button in case you click by accident on a note or in case you just want to check the content of the note